tests/test_hodgkin.py

```diff
@@ -240,3 +242,21 @@ def test_evaluation(self, graph, model):
             expected = evaluated_derivatives[state_symbol.name]
             actual = evaluated_deriv
             assert float(actual) == pytest.approx(expected)
+
+    def test_get_symbol_by_ontology_term(self, graph, model):
```

Comment:
Why is this unrelated test in the PR? (Not that I have an issue with the test itself!)

Reply:
because it's a (previously untested) method that the other test uses, the other test is needed to ensure 100% coverage and I thought it would be neater to also have a test for this method rather than relying on it failing in the test for get_equations_for